modules/upgrade/pages/server-major-upgrade-uyuni.adoc

```diff
@@ -0,0 +1,102 @@
+[server-major-upgrade]
+= Server - Major Upgrade
+
+If {productname} core compontents are upgraded to new major versions, {productname} Server upgrades are considered as major.
+This is the case, if a version upgrade of PostgreSQL, Salt, or openSUSE Leap is needed.
+openSUSE Leap is the underlying base operating system (OS).
+For background information about upgrading openSUSE Leap, see the Startup Guide at https://doc.opensuse.org.
+
+[IMPORTANT]
+====
+For more information, see the release notes at https://www.uyuni-project.org/pages/stable-version.html.
+Reading the release notes is critical.
+Sometimes additional manual steps are required.
+====
+
+
+Run the [command]``/usr/lib/susemanager/bin/server-migrator.sh`` script to upgrade the base OS and {productname} Server.
+
+[NOTE]
+====
+Before running the [command]``server-migrator.sh`` script, check whether the most recent version of the [package]``susemanager`` package is installed:
```

Comment:
I would take this info out of the admon, put it above the 'run the script` instruction, and turn it into a proper procedure.

Reply:
ok